frontend/src/pages/spotnet/dashboard/Dashboard.jsx

```diff
@@ -189,11 +189,7 @@ export default function Component({ telegramId }) {
             >
               {isClosing ? 'Closing...' : 'Redeem'}
             </Button>
-            {closePositionError && (
-              <div className="error-message">
-                Error: {closePositionError.message} <AlertHexagon className="form-alert-hex" />
-              </div>
-            )}
+            {closePositionError && (notify("Error: " + closePositionError.message))}
```

Comment:
actually notify can be used inside `useClosePosition` hook, in `onError` callback, remove it from here.